### PR TITLE
Set babushka's name and version on every new connection.

### PR DESCRIPTION
**Pull request by nihohit**

### CODE REVIEW

File: redis/src/connection.rs [hunk 1]
```diff
@@ -760,6 +760,23 @@ pub fn connect(
...
 }
 
+pub(crate) fn client_set_info_pipeline() -> Pipeline {
+    let mut pipeline = crate::pipe();
+    pipeline
+        .cmd("CLIENT")
+        .arg("SETINFO")
+        .arg("LIB-NAME")
+        .arg(std::env!("BABUSHKA_NAME"))
+        .ignore();
```
Comment (barshaul) [inline redis/src/connection.rs:770]: What happens in redis < 7? does it fail in silent?

Comment (barshaul) [reply redis/src/connection.rs:770]: also - I think we should add a comment stating that it works only for >= V7

Comment (nihohit) [reply redis/src/connection.rs:770]: > What happens in redis < 7? does it fail in silent?
yes. I'll add a comment.

Comment (nihohit) [reply redis/src/connection.rs:770]: it's supposed to fail silently, and there's a comment saying that.

**Review by barshaul — APPROVED** [review]

**Pull request merged**